arm_robots/src/arm_robots/panda.py

```diff
@@ -15,11 +21,111 @@ def __init__(self, robot_namespace: str = 'combined_panda', force_trigger: float
                                     arms_controller_name='effort_joint_trajectory_controller',
                                     force_trigger=force_trigger,
                                     **kwargs)
+        self.nebula_arm = "nebula_arm"
+        self.nebula_id = "panda_1"
+        self.rocket_arm = "rocket_arm"
+        self.rocket_id = "panda_2"
+        self.nebula_wrist = "panda_1_link8"
+        self.rocket_wrist = "panda_2_link8"
+        self.nebula_gripper = PandaGripper(self.robot_namespace, self.nebula_id)
+        self.rocket_gripper = PandaGripper(self.robot_namespace, self.rocket_id)
+        rospy.Subscriber("/"+self.robot_namespace+"/"+self.nebula_id+"_state_controller/franka_states", FrankaState, self.nebula_error_cb)
```

Comment:
used ns_join instead of string concatenation. Also are you use we want the leading "/"? That would make it impossible for anyone to use ROS_NAMESPACE to namespace this topic.